install.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+LOG_START='\n\e[1;36m' # new line + bold + color
+LOG_END='\n\e[0m' # new line + reset color
+
+WORKDIR=$PWD
+
+printf "${LOG_START}Installing repositories...${LOG_END}"
+
+./install-repositories.sh
+
+cd keep-subgraph
+
+printf "${LOG_START}Running subgraph scripts...${LOG_END}"
+
+CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY=${CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY} ETH_RPC_URL=${ETH_RPC_URL} SUBGRAPH_DEPLOY_KEY=${SUBGRAPH_DEPLOY_KEY} SUBGRAPH_SLUG=${SUBGRAPH_SLUG} ./deploy-subgraph.sh
+
+printf "${LOG_START}Starting dApp...${LOG_END}"
+
+cd ${WORKDIR}
+
+./run-dapp-locally.sh
```

Comment:
Shouldn't this script execute `./install-dapp.sh`?

Reply:
We don't need to run `yarn start` for `https://allthekeeps.test.keep.network/` and that's why I separated `install-dapp.sh` and `run-dapp-locally.sh`. `install.sh` was used for local development hence `run-dapp-locally.sh` is used here. I guess `install.sh` can be renamed to smth like `install-locally.sh`, but that's probably in a seperate PR as per [this](https://github.com/keep-network/allthekeeps/pull/3#discussion_r702938384) comment?